pystac/collection.py

```diff
@@ -372,29 +436,44 @@ class Provider:
             licensor, producer, processor or host.
         url : Optional homepage on which the provider describes the dataset
             and publishes contact information.
-
-    Attributes:
-        name : The name of the organization or the individual.
-        description : Optional multi-line description to add further provider
-            information such as processing details for processors and producers,
-            hosting details for hosts or basic contact information.
-        roles : Optional roles of the provider. Any of
-            licensor, producer, processor or host.
-        url : Optional homepage on which the provider describes the dataset
-            and publishes contact information.
+        extra_fields : Optional dictionary containing additional top-level fields
+            defined on the Provider object.
+    object.
```

Comment:
Is this line a dangling paste/reformat? Sometimes I can't be sure in Github but it looks weird.

Reply:
Yeah, definitely a typo. Fixed in b0f9b496128f4ad50909812f8193401ccad9a924.